packages/e2e-test-app/test/__snapshots__/ViewComponentTest.test.ts.snap

```diff
@@ -520,6 +520,22 @@ exports[`ViewTests Views can have overflow 1`] = `
             },
           ],
         },
+        {
+          "Background": null,
```

Comment:
Please note, these snapshots changes are expected. There was a long standing undetected discrepancy in the overflow example in RNTester > View Example where borders were not rendered.